src/screens/WebsocketResearchScreen/SampleChatInfoScreen.tsx

```diff
@@ -87,8 +100,7 @@ export const styles = StyleSheet.create({
   containerGroupName: {
     flexDirection: 'row',
     justifyContent: 'space-between',
-    alignItems: 'center',
-    paddingLeft: 20
+    alignItems: 'center'
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The `paddingLeft` property has been removed from the `containerGroupName` style. Make sure this doesn't affect the layout in an undesired way.



<!-- This is an auto-generated comment by OSS CodeRabbit -->